_docs/technical/2017-02-14-leader-selection.md

```diff
@@ -0,0 +1,39 @@
+---
+layout: default
+title: Leader Selection in Cardano SL
+permalink: /technical/leader-selection/
+group: technical
+visible: true
+---
+[//]: # (Reviewed at e1d0f9fb37a3f1378341716916f0321fb55698df)
+
+# Leader Selection in Cardano SL
+
+This guide describes slot-leader selection process.
+
+## Follow the Satoshi
+
+As mentioned [earlier](/cardano/proof-of-stake/#follow-the-satoshi),
+Cardano SL uses Follow the Satoshi (FTS) algorithm to choose slot-leaders. Slot-leaders for the
+current epoch (i.e. for each slot of the current epoch) are computed by FTS in the beginning of current epoch.
+So genesis block contains a list of selected slot-leaders.
+
+FTS uses a shared seed which is result of [Multi Party Computation (MPC)](/cardano/proof-of-stake/#multi-party-computation)
+algorithm for previous epoch: in the result of MPC some nodes reveal their seeds, xor of these seeds is called shared seed.
+
+Choose several random stakeholders (specifically, their amount is
+currently hardcoded in 'Pos.Constants.epochSlots').
```

Comment:
I think it's better to add links to code when modules or functions of fields or anything else is mentioned